.travis.yml

```diff
@@ -25,9 +27,8 @@ install:
   # Ref build: https://travis-ci.com/sql-machine-learning/sqlflow/builds/107870583
   - echo "skip install"
 
-# NOTE: mount all $GOPATH into container, then after build and test,
+# NOTE(weiguoz): mount $GOPATH into container, then after build and test,
```

Comment:
Why note me : )

Reply:
I checked the git history. This line is introduced at https://github.com/sql-machine-learning/sqlflow/pull/319/files.